Extension/src/LanguageServer/extension.ts

```diff
@@ -536,6 +537,10 @@ function selectDefaultCompiler(sender?: any): void {
     clients.ActiveClient.promptSelectCompiler(true, sender);
 }
 
+function selectIntelliSenseConfiguration(sender?: any): void {
+    clients.ActiveClient.promptSelectIntelliSenseConfiguration(true, sender);
```

Comment:
Shouldn't all(?) of these commands be waiting until the language service is ready before proceeding?  In the case of this function, we might now have received the compiler defaults from the native side until then.

Reply:
Not sure, I was just following the existing code for promptSelectCompiler.

Reply:
`promptSelectCompiler` is definitely wrong.  I had to add `notifyWhenLanguageClientReady` there in the `update-walkthrough` branch, otherwise use of that command from the walkthrough would easily fail whenever clicked when the extension was not yet fully started up.

Reply:
ok